src/frontend/src/catalog/system_catalog/pg_catalog/pg_matviews.rs

```diff
@@ -16,15 +16,18 @@ use risingwave_common::types::DataType;
 
 use crate::catalog::system_catalog::SystemCatalogColumnsDef;
 
-/// The catalog `pg_matviews_info` contains the information about the matviews.
-pub const PG_MATVIEWS_INFO_TABLE_NAME: &str = "pg_matviews_info";
-pub const PG_MATVIEWS_INFO_COLUMNS: &[SystemCatalogColumnsDef<'_>] = &[
-    (DataType::Int32, "matviewid"),
+/// The view `pg_matviews` provides access to useful information about each materialized view in the
+/// database.
+/// Ref: [`https://www.postgresql.org/docs/current/view-pg-matviews.html`]
+pub const PG_MATVIEWS_TABLE_NAME: &str = "pg_matviews";
+pub const PG_MATVIEWS_COLUMNS: &[SystemCatalogColumnsDef<'_>] = &[
```

Comment:
The reason why using `pg_matviews_info` rather than `pg_matviews` here is that it contains a different column `matviewgraph`, which is different from pg and there may be problems in integrating some tools. And we'd better not change it, because the cloud team is already using this table. see: https://github.com/risingwavelabs/risingwave-cloud/blob/9abad797b1c9abda89d27d26505c79433fb3c417/mgmt/internal/model/mat_view.go

Reply:
Or get them to update it too?

Reply:
Ok, I'd prefer to contact the cloud team to update it